test/cpp/jit/test_gpu.cpp

```diff
@@ -6747,6 +6747,88 @@ TEST(NVFuserTest, FusionGridReduction8_CUDA) {
   testValidate(&fusion, out, {input}, {aten_output}, __LINE__, __FILE__);
 }
 
+TEST(NVFuserTest, FusionGridReduction9_CUDA) {
```

Comment:
I think it's also good to have tests for grid broadcast.

Reply:
Grid persistence tests are testing grid broadcast, since we need a reduction before the broadcast to trigger it.

Reply:
Do you want more grid broadcast tests?

Reply:
I was thinking of a bare minimum example, but `FusionGridPersistence` seems to be the simplest case. Looks sufficient to me.